schema/codypro_util.go

```diff
@@ -0,0 +1,21 @@
+package schema
+
+// FrontendCodyProConfig defines the subset of Cody Pro-specific configuration data
+// that we make available to the Sourcegraph frontend. (So any sensitive or irrelevant
+// site configuration data doesn't blindly get exposed to the web frontend.)
+//
+// Changes to this type need to be mirrored in `jscontext.ts`.
```

Comment:
Clarifying the path seems more helpful because we have two more `jscontext.ts` files in the repo:
```suggestion
// Changes to this type need to be mirrored in `client/web/src/cody/jscontext.ts`.
```